otter/integration/tests/test_convergence.py

```diff
@@ -129,6 +178,16 @@ def _choose_half_the_servers(self, (code, response)):
         self.n_killed = self.n_servers / 2
         return ids[:self.n_killed]
 
+    def _choose_random_servers(self, (code, response), n):
+        """Selects ``n`` randomly selected servers from those returned by the
+        ``get_scaling_group_state`` function.
+        """
+
+        if code != 200:
+            raise Exception("Got 404; dude, where's my scaling group?")
+        ids = map(lambda obj: obj['id'], response['group']['active'])
```

Comment:
Just a python style thing: list comprehensions are often more concise than `map` when specifically used in conjunction with `lambda`. I am all for maps and lambdas, but consider:

```
[obj['id'] for obj in response['group']['active']
```
vs
```
map(lambda obj: obj['id'], response['group']['active'])
```

Please just consider this comment as informative. Conciseness isn't everything, anyway.